internal/graphql/graphql.go

```diff
@@ -12,16 +12,15 @@ import (
 	"github.com/mensatt/backend/internal/graphql/directives"
 	"github.com/mensatt/backend/internal/graphql/gqlserver"
 	"github.com/mensatt/backend/internal/graphql/resolvers"
-	"github.com/mensatt/backend/pkg/imageuploader"
 	"github.com/mensatt/backend/pkg/utils"
 )
 
 type GraphQLParams struct {
-	DebugEnabled  bool
-	Database      *ent.Client
-	JWTKeyStore   *utils.JWTKeyStore
-	ImageUploader *imageuploader.ImageUploader
-	ImageBaseURL  string
+	DebugEnabled bool
+	Database     *ent.Client
+	JWTKeyStore  *utils.JWTKeyStore
+	ImageAPIURL  string
+	ImageAPIKey  string
```

Comment:
Might just be a naming thing, but why are those two values part of the "GraphQLParams"? What effect does the image service have on GQL? Imo it should be none.

Reply:
Need to pass down config values, probably a better way to do it but this way works